### PR TITLE
Jj gnisci

### PR DESCRIPTION
**Pull request by gnizla**
- I have changed the dating suggested by SH for b4 from 1600-1800 to 1750-1850 on the basis of miniatures
- There are a lot of miniatures in b. 4! The identification of d. 36, d. 37, and d. 38 is a bit tentative in case you have any insight.
- I am a bit uncertain about how to visualize layout, this is something that can be resolved further down the line though.

### CODE REVIEW

**Review by MassimoVilla — CHANGES_REQUESTED** [review]
Thank you Jacopo! I have just a couple of trivial remarks, perhaps typos.

File: OxfordBodleian/Juel-Jensen/BDLaethb4.xml [hunk 1]
```diff
@@ -1308,32 +1308,365 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
+      The miniature features three episodes. First, <persName ref="PRS13193Giyorgis">St George</persName>
+         has a vision of <persName ref="PRS5684JesusCh"/> in a cloud; then, he is mauled by men holding adzes;
+         and, finally, his head is set ablaze (full-page).
+   </desc>
+</decoNote>
+
+<decoNote type="miniature" xml:id="d10">
+   <locus target="#42r"/>
+   <desc>
+      Above, two tortures hit <persName ref="PRS13193Giyorgis">St George</persName>
```
[the remark this answers is not in the document]
Comment (gnizla) [reply OxfordBodleian/Juel-Jensen/BDLaethb4.xml:1416]: Yes, well spotted, will change.

File: OxfordBodleian/Juel-Jensen/BDLaethb4.xml [hunk 2]
```diff
@@ -1308,32 +1308,365 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
+      confronts the king's four evil attendants (full-page).
+   </desc>
+</decoNote>
+
+<decoNote type="miniature" xml:id="d14">
+   <locus target="#50v"/>
+   <desc>
+     There are three episodes. Above, <persName ref="PRS5684JesusCh"/> has brought <persName ref="PRS13193Giyorgis">St George</persName>
+       back to life. Below, to the left, St George baptizes the four jailers turned Christian after his miracolous recovery
+       in a pool at the foot of a hill surmounted by a Gondarene castle; to the right, an executioner has cut off their heads (full-page).
```
[the remark this answers is not in the document]
Comment (gnizla) [reply OxfordBodleian/Juel-Jensen/BDLaethb4.xml:1452]: Both terms are attested in the literature :)

**Review by thea-m — CHANGES_REQUESTED** [review]
Thank you! One extremely minor remark. 
I corrected some typos directly in the files (also old ones in other parts of the description). 
I do join Massimo in his Gondarene/Gondarine question: Both are used in our records, to me it seems better to agree on one (I have no opinion which).

**Review by gnizla — COMMENTED** [review]
Thanks for comments!

**Comment by gnizla** [comment]
I do join Massimo in his Gondarene/Gondarine question: Both are used in our records, to me it seems better to agree on one (I have no opinion which).

Gondarine is more common than Gondarene so I guess it would make more sense to go for that if we want standardization. @sologebre any opinion?

**Pull request merged**